### ISSUE TITLE
Is it possible to autofix certain problems?

### ISSUE THREAD

**Issue by alessandro-newzoo** [monosans/djlint-vscode#115]
Hello! Thanks for this beautiful extension :)

I was wondering if there's a way to enable auto fixing of certain problems like for example the two mentioned below?
It does work for some problems _(like adding spaces inside a variable)_, but it'd be really nice if it could work with these too.

## Example

Given this code:

```twig
{% block name %}
   {% include 'header.twig' %}
{% endblock %}
```

These problems are reported in the Problems panel, based on my .djlintrc config:
- ```Endblock should have name. Ex: {% endblock body %}. {% endblock %} (T003)```
- ```Double quotes should be used in tags. {% include 'header.t (T002)```

And ideally the code above should be replaced with the following on save:
```twig
{% block name %}
   {% include "header.twig" %}
              ^           ^
{% endblock name %}
            ^^^^
```

Or perhaps this is something I should request in the Djlint repo?

Thank you! ❤️

**Comment by alessandro-newzoo** [comment]
oh okay, sorry wasn't sure which part handled these things.
Thanks!
